### PR TITLE
rules: Include rules from xmas promotion

### PR DESCRIPTION
**Pull request by charlieegan3**

### CODE REVIEW

File: bundle/regal/rules/idiomatic/circular_import_test.rego [hunk 1]
```diff
@@ -0,0 +1,265 @@
+package regal.rules.idiomatic["circular-import_test"]
```
Comment (anderseknert) [inline bundle/regal/rules/idiomatic/circular_import_test.rego:1]: I suppose avoiding this is idomatic, but I wonder if the `imports` category would be even more appropriate?

File: bundle/regal/rules/style/rule_name_repeats_package_test.rego [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+	r == {
+		"policy_foo_bar",
+		"foo_bar",
+		"bar",
+		"policyFooBar",
+		"fooBar",
+	}
+}
+
+test_rule_empty_if_no_repitition if {
```
Comment (anderseknert) [inline bundle/regal/rules/style/rule_name_repeats_package_test.rego:37]: repetition ... this is consistently misspelled here hehe

File: docs/rules/style/rule-name-repeats-package.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+This rule was inspired by [Go Code Review Comments](https://github.com/golang/go/wiki/CodeReviewComments#package-names).
+
+## Configuration Options
+
+This linter rule provides the following configuration options:
+
+```yaml
+rules:
+  community:
```
Comment (anderseknert) [inline docs/rules/style/rule-name-repeats-package.md:45]: style

File: docs/rules/style/rule-name-repeats-package.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
+# rule-name-repeats-package
+
+**Summary**: Avoid repeating package path in rule names
+
+**Category**: Community
```
Comment (anderseknert) [inline docs/rules/style/rule-name-repeats-package.md:5]: Style

**Review by anderseknert — COMMENTED** [review]
Awesome! Just some nits :)

File: bundle/regal/rules/idiomatic/circular_import_test.rego (hunk 1 above)
Comment (charlieegan3) [reply bundle/regal/rules/idiomatic/circular_import_test.rego:1]: yeah I can move it to there!

File: docs/rules/style/rule-name-repeats-package.md (hunk 3 above)
Comment (charlieegan3) [reply docs/rules/style/rule-name-repeats-package.md]: Should be all fixed in https://github.com/StyraInc/regal/pull/558/commits/c628729de019281fab625f49ebcf0b3e67294db9

File: bundle/regal/rules/idiomatic/circular_import_test.rego (hunk 1 above)
Comment (charlieegan3) [reply bundle/regal/rules/idiomatic/circular_import_test.rego]: Moved in https://github.com/StyraInc/regal/pull/558/commits/c628729de019281fab625f49ebcf0b3e67294db9

**Review by anderseknert — APPROVED** [review]
Very nice!

**Pull request merged**